lib/beaker-pe/install/pe_utils.rb

```diff
@@ -578,6 +583,32 @@ def simple_monolithic_install(master, agents, opts={})
           end
         end
 
+
+        def config_master_for_proxy_access
+          step "configuring master to use proxy" do
+            @master_ip = on master, "hostname -I | tr '\n' ' '"
+            on master, "echo \"#{@proxy_ip.stdout}  #{@proxy_hostname}\" >> /etc/hosts"
+            on master, "echo \"#{@master_ip.stdout}  #{master.connection.vmhostname}\" >> /etc/hosts"
+            on master, "echo \"10.32.77.209    osmirror.delivery.puppetlabs.net\" >> /etc/hosts"
+            on master, "iptables -A OUTPUT -p tcp -d #{master.connection.vmhostname} -j ACCEPT"
+            on master, "iptables -A OUTPUT -p tcp -d 10.240.0.10 -j ACCEPT"
```

Comment:
Is there another method we can use in place of hard coding IP Addresses? I'm concerned about how this might break if the IP should change.

Reply:
that should be fixed now